examples/disaggregated/slurm/submit.sh

```
@@ -1,37 +1,41 @@
-#   gen_enable_attention_dp=${10}
-#   gen_gpu_memory_fraction=${11}
-#   eplb_num_slots=${12}
-#   mtp_size=${13}
-#   concurrency=${14}
+ctx_num=1
+concurrency=8
+isl=1024
+osl=1024
+multi_round=10
+streaming=true
+container_image=<container_image>
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)